### PR TITLE
Add ytop

### PR DESCRIPTION
**Pull request by openaphid**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -311,6 +311,7 @@ See also [A comparison of operating systems written in Rust](https://github.com/
 * [uutils/coreutils](https://github.com/uutils/coreutils) — A cross-platform Rust rewrite of the GNU coreutils [<img src="https://api.travis-ci.org/uutils/coreutils.svg?branch=master">](https://travis-ci.org/uutils/coreutils)
 * [watchexec](https://github.com/watchexec/watchexec) — Executes commands in response to file modifications [<img src="https://api.travis-ci.org/watchexec/watchexec.svg?branch=master">](https://travis-ci.org/watchexec/watchexec)
 * [XAMPPRocky/tokei](https://github.com/XAMPPRocky/tokei) — counts the lines of code [<img src="https://api.travis-ci.org/XAMPPRocky/tokei.svg?branch=master">](https://travis-ci.org/XAMPPRocky/tokei)
+* [ytop](https://github.com/cjbassi/ytop) - A TUI system monitor written in Rust
```
Comment (palfrey) [inline README.md]: It has a Github actions CI build. Can you please add the build badge for that to this?

Comment (openaphid) [reply README.md]: Sure. I didn't notice that it's using Github actions CI to replace Travis CI.

Comment (palfrey) [reply README.md]: Thanks for adding the build badge, but it doesn't have the build branch listed, which means we may in the future get "build failed" even if the main branch is working. Can you please change it so it refers to the master branch please?

Comment (openaphid) [reply README.md]: I should have noticed it when the Travis checker failed. Now the branch information is added but the checker failed due to network errors. Is it possible to rerun it without making another commit?

Comment (palfrey) [reply README.md]: Rebuilding now

**Pull request merged**